components/concept/sync/src/main/java/mozilla/components/concept/sync/OAuthAccount.kt

```diff
@@ -56,24 +56,24 @@ interface AccountObserver {
     /**
      * Account just got logged out.
      */
-    fun onLoggedOut()
+    fun onLoggedOut() = Unit
 
     /**
      * Account was successfully authenticated.
      * @param account An authenticated instance of a [OAuthAccount].
      */
-    fun onAuthenticated(account: OAuthAccount)
+    fun onAuthenticated(account: OAuthAccount, newAccount: Boolean = false) = Unit
```

Comment:
Also: missing `@param newAccount` docs.

Reply:
Also: I'd prefer you omit a default `false` value, so that we're forced to change every invocation of `onAuthenticated`. This way it's super obvious we're not missing any calls, and at every call site you immediately see what kind of notification we'll fire. Default params are a footgun.